src/serlio/modifiers/MayaCallbacks.h

```diff
@@ -33,6 +33,8 @@
 #include <string>
 #include <vector>
 
+using CGACErrors = std::vector<std::pair<prt::CGAErrorLevel, std::wstring>>;
+
 class MayaCallbacks : public IMayaCallbacks {
```

Comment:
please use this opportunity to move all function implementations in header to the cpp file

Reply:
moved all function implementations to cpp file.